app/src/main/java/com/android/sample/MainActivity.kt

```diff
@@ -66,8 +70,6 @@ fun PlateSwipeApp() {
         route = Route.SWIPE,
     ) {
       composable(Screen.SWIPE) { SwipePage(navigationActions, recipesViewModel) }
-      composable(Screen.OVERVIEW_RECIPE) { RecipeOverview(navigationActions, recipesViewModel) }
```

Comment:
Could you please clarify why these two screens were removed from the Screen Swipe?

Thank you!

Reply:
You have good eyes! Sorry I wrongly deleted those lines. Again thank you for pointing me this